loadtest/control/actions.go

```diff
@@ -215,6 +215,40 @@ func CreatePost(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("post created, id %v", postId)}
 }
 
+// CreatePostReply replies to a randomly picked post.
+func CreatePostReply(u user.User) UserActionResponse {
+	team, err := u.Store().RandomTeamJoined()
+	if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+	channel, err := u.Store().RandomChannelJoined(team.Id)
+	if errors.Is(err, memstore.ErrChannelStoreEmpty) {
+		return UserActionResponse{Info: "no channels in store"}
+	} else if err != nil {
+		return UserActionResponse{Err: NewUserError(err)}
+	}
+
+	post, err := u.Store().RandomPostForChannel(channel.Id)
```

Comment:
I wonder if can get a weighted result for more recent posts? It would be much more realistic.

Reply:
Ideally yes. I think the solution is to make sure the store has more recent posts.

There are a lot of details like this we can improve but first I want to know how much improving those will affect the overall result because the ultimate goal here is not simulation but load-testing. 

So if we can stress the target instance the same way (on average) that a user does I think we are good. Going into more details might be a fun experiment but it can get hairy pretty fast.